### PR TITLE
[547] Allow visa to be updated for scheduled

### PR DESCRIPTION
**Pull request by darokel**
### Context

- https://trello.com/c/fD07l44U/547-add-change-links-to-update-funding-visa-for-scheduled

### Changes proposed in this pull request

- Allows providers to update the visa settings for scheduled courses

### Guidance to review

- Check you can change these settings for scheduled courses

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased main
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

**Comment by darokel** [comment]
[Add change links to update funding/visa for scheduled](https://trello.com/c/fD07l44U/547-add-change-links-to-update-funding-visa-for-scheduled)

**Review by defong — CHANGES_REQUESTED** [review]
I think we need clarification since `scheduled` is essentially `published`, which undermines enforcing items required.

**Comment by defong** [comment]
See https://github.com/DFE-Digital/publish-teacher-training/pull/3025/commits/abb3b369f4e62e396b3a8b9e7048fb72b05f2ce4

File: app/policies/course_policy.rb [hunk 1]
```diff
@@ -38,7 +38,7 @@ def send_vacancies_updated_notification?
   end
 
   def can_update_funding_type?
-    course.draft_or_rolled_over?
+    course.changeable?
```
Comment (defong) [inline app/policies/course_policy.rb:41]: We can revert this as well.

Comment (darokel) [reply app/policies/course_policy.rb]: done

**Review by defong — APPROVED** [review]

**Pull request merged**